src/main/groovy/com/rundeck/plugins/ansible/ansible/AnsibleVault.java

```diff
@@ -26,19 +27,26 @@ public class AnsibleVault {
 
     public final String ANSIBLE_VAULT_COMMAND = "ansible-vault";
 
+    private ProcessExecutor.ProcessExecutorBuilder processExecutorBuilder;
```

Comment:
Do we want this to be re-used across invocations of methods? Perhaps this should be an instance variable in each method?

Reply:
I added that to mock it in the test